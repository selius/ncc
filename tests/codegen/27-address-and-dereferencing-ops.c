int main()
{
	int *a;
	int b;
	int c;

	float *d;
	float e;

	a = &b;
	b = 7;
	__print_int(*a);

	c = 10;
	a = &c;
	__print_int(*a);

	d = &e;
	e = 56.34;
	__print_float(*d);

	__print_float(*&e);

	return 0;
}
