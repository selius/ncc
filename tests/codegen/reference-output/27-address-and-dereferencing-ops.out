7
10
56.340000
56.340000
